management/storageservice/client.go

```diff
@@ -128,14 +128,14 @@ func (self StorageServiceClient) GetBlobEndpoint(storageService *StorageService)
 	return "", errors.New(fmt.Sprintf(errBlobEndpointNotFound, storageService.ServiceName))
 }
 
-func (self *StorageServiceClient) createStorageServiceDeploymentConf(name, location string) StorageServiceDeployment {
+func (self *StorageServiceClient) createStorageServiceDeploymentConf(name, location string, accountType AccountType) StorageServiceDeployment {
```

Comment:
-1 for this helper method's existence as well. Ideally user should provide this or we should have one helper like `func NewCreateStorageAccountParameters(name, location string) CreateStorageAccountParameters` and pass that to `CreateStorageService()`